src/config/server.rs

```diff
@@ -16,7 +16,7 @@ pub struct Server {
   pub enable_response_validation: Option<bool>,
   pub global_response_timeout: Option<i64>,
   pub port: Option<u16>,
-  #[serde(default, skip_serializing_if = "is_default")]
+  #[serde(skip)]
   pub upstream: Upstream,
```

Comment:
I think we can just delete this field.

Reply:
I was thinking to not touch the logic of request handling upstrem in Server struct. So I only filled in the upstream field of Server from new upstream directive. If we are gonna delete the upstream filed then I will move to Config’filed thay is same level with Server field in config. then also refactor the upstream request to this new upstream filed. what do yoy think ?

Reply:
Yeah, I think that's the way we should go about this.

Reply:
@tusharmath 
Done! Help review, thanks.